rfb/rfbproto.h

```diff
@@ -91,14 +91,15 @@ typedef int8_t rfbBool;
 #endif
 
 #define rfbMax(a,b) (((a)>(b))?(a):(b))
-#if !defined(WIN32) || defined(__MINGW32__)
+#if !defined(WIN32)
 #ifdef LIBVNCSERVER_HAVE_SYS_TIME_H
 #include <sys/time.h>
 #endif
 #ifdef LIBVNCSERVER_HAVE_NETINET_IN_H
 #include <netinet/in.h>
 #endif
 #define SOCKET int
```

Comment:
I think a cleaner way would be to use `typedef SOCKET sock_t` for windows and `typedef int sock_t` for linux. Uppercase types look weird.)))

Reply:
I refactored the patch to use `rfbSocket` and `RFB_INVALID_SOCKET` :-)